chapters/chap02-sec05.tex

```diff
@@ -68,5 +69,27 @@ \section{Processing $\zethTx$}\label{zeth-protocol:process-tx}
                     \item $\mixer.\balance = \mixer.\balance - \vout$
                 \end{itemize}
             \item Emit an event (\cref{sssec:ethereum-events}) $\evMixOut$ of type \mixEventDType, containing the new root $\mkroot'$ of the Merkle tree of commitments, the nullifiers $\indexedset{\nf{i}}{i \in [\jsin]}$, commitments to the newly created \zethnotes $\inp{\mix}.\primInp.\cms{}$, and the corresponding ciphertexts $\inp{\mix}.\primInp.\ciphers$.
-        \end{itemize}
+        \end{enumerate}
 \end{enumerate}
+
+\begin{remark}\label{zeth-protocol:process-tx:dispatch-call}
```

Comment:
What do you think about moving the remarks related to the "Zecale integration" into a "Zecale box" (in a similar way as we have \notebox)?

Reply:
As a matter of preference I would probably favour a formal remark over a coloured box, but I accept that some of this zecale-specific content can feel a bit out of place.  I think I mentioned previously that we could consider just dropping one of the remarks to keep everything contained in a single set of paragraphs.

One approach could be to move all zecale-related content to a separate section / appendix, and simply refer to the relevant places in the document where modifications would be required.

Reply:
> As a matter of preference I would probably favour a formal remark over a coloured box

Sure, no problem, that's fine how it is (though I don't think a "box" is less formal than a remark, it "sandboxes" the content from the main body ;) ).

> One approach could be to move all zecale-related content to a separate section / appendix, and simply refer to the relevant places in the document where modifications would be required. Let me know what you think

That's fine, let's keep it like that for now